pkg/apis/pipeline/v1alpha1/build_gcs_resource.go

```diff
@@ -177,3 +177,7 @@ func getArtifactType(val string) (GCSArtifactType, error) {
 func (s *BuildGCSResource) GetUploadVolumeSpec(spec *TaskSpec) ([]corev1.Volume, error) {
 	return getStorageUploadVolumeSpec(s, spec)
 }
+
+func (s *BuildGCSResource) GetDownloadVolumeSpec(spec *TaskSpec) ([]corev1.Volume, error) {
+	return getStorageUploadVolumeSpec(s, spec)
```

Comment:
v minor but would renaming to `getStorageTransferVolumeSpec` or maybe even just `getStorageVolumeSpec` or similar make sense given this is now also being used for the download volume?